pkg/github/github_test.go

```diff
@@ -36,7 +37,17 @@ func (client *MockClient) Get(url string) ([]byte, error) {
 	if strings.Contains(url, "compare") {
 		return httpClient.Get("file://./testdata/compare_v3.json")
 	} else if strings.Contains(url, "CHANGELOG") {
+		if strings.Contains(url, "real_release_branch") {
+			// We are running a test where the repo has a release branch
+			return httpClient.Get("file://./testdata/release_branch_changelog.md")
+		}
 		return httpClient.Get("file://./testdata/simple_changelog.md")
+	} else if strings.Contains(url, "branches") {
+		if strings.Contains(url, "real_release_branch") {
+			// We are running a test where the repo has a release branch
+			return httpClient.Get("file://./testdata/branch_v3.json")
+		}
+		return nil, fmt.Errorf("Not found")
```

Comment:
Error here should probably be a bit more descriptive

Reply:
but why, it is a fake error. also, this is very nearly the real error github gives:
```
{
  "message": "Branch not found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#get-a-branch"
}
```

Reply:
changed to "Branch not found"